include/beast/websocket/detail/invokable.hpp

```diff
@@ -125,15 +125,17 @@ class invokable
     void
     emplace(F&& f);
 
-    void
+    bool
     maybe_invoke()
     {
         if(base_)
         {
             auto const basep = base_;
```

Comment:
Can this local be avoided?
`        if (! base_)
            return false;
        (*base_)();
        base_ = nullptr;
        return true;`